meta/Parameters.m

```diff
@@ -169,7 +169,24 @@
 
 FillInputParametersFromTuples::usage="";
 
-DecreaseIndexLiterals::usage="";
+DecreaseIndexLiterals::usage="@note
+Definetly safe calls:
+	f[ { Head1[Int1] , Head2[Int2], ... } ]
+	f[ { Head1[Int1] , Head2[Int2], ... }, { HeadI1, HeadI2, ...} ]
+Possible calls:
+	1. f[ exprD ]
+	2. f[ exprD, listH ]
+where
+	#1 exprD is @usomething which probably contains subexpressions (at @any level) of the form HeadOfExpression[SomeInteger]
+	#2 listH is List which has subexpressions (only at @lowest level) of the form	HeadOfExpression
+Output:
+	1. Gets names ({HeadOfExpression1, HeadOfExpression2, ...}) of all InputParameters, ExtraParameters, allModelParameters, allOutputParameters and changes
+	   in exprD any HeadOfExpression1[SomeInteger] to HeadOfExpression1[SomeInteger-1]
+	2. Gets names ({HeadOfExpression1, HeadOfExpression2, ...}) in listH and changes
+	   in exprD any HeadOfExpression1[SomeInteger] to HeadOfExpression1[SomeInteger-1]
+Issues:
```

Comment:
Can you please create a github issue for this defect?

Reply:
Done in #152.